src/libsystemd/sd-network/sd-network.c

```diff
@@ -212,6 +212,10 @@ _public_ int sd_network_link_get_required_operstate_for_online(int ifindex, char
         return 0;
 }
 
+_public_ int sd_network_link_get_activation_policy(int ifindex, char **policy) {
+        return network_link_get_string(ifindex, "ACTIVATION_POLICY", policy);
```

Comment:
hmm, if the field is missing I'd synthesize "UP" here, i.e. the only supported policy in the status quo ante.

i mean, this doesn't matter, as we don't really support mixing client and server, but it might still be nice to add, because it's easy

Reply:
ack, i'll update in the next push, thanks